test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -194,26 +194,45 @@ func schedulePods(config *testConfig) int32 {
 	config.podCreator.CreatePods()
 
 	prev := 0
+	// On startup there may be a latent period where NO scheduling occurs (qps = 0).
+	// We are interested in low scheduling rates (i.e. qps=2),
 	minQps := int32(math.MaxInt32)
 	start := time.Now()
+
+	// Bake in time for the first pod scheduling event.
+	waitForStart := func() {
```

Comment:
just for readability.  IMO `waitForStart` in one line proceeding the "actual" test is instantly readable even if you've never seen the code before.